learning_resources_search/serializers.py

```diff
@@ -0,0 +1,133 @@
+"""Serializers for opensearch data"""
+# pylint: disable=unused-argument,too-many-lines
+import logging
+
+from rest_framework import serializers
+
+from learning_resources.models import Course, Program
+from learning_resources.serializers import (
+    LearningResourceRunSerializer,
+    LearningResourceSerializer,
+)
+from learning_resources_search.api import (
+    gen_course_id,
+    gen_program_id,
+)
+
+log = logging.getLogger()
+
+
+class OSLearningResourceRunSerializer(LearningResourceRunSerializer):
+    instructors = serializers.SerializerMethodField()
+
+    def get_instructors(self, instance):
+        """
+        Get a list of instructor names for the course run
+        """
+        return [
+            (instructor.full_name or f"{instructor.first_name} {instructor.last_name}")
+            for instructor in instance.instructors.all()
+        ]
+
+
+class OSLearningResourceSerializer(LearningResourceSerializer):
+    """Serializer for LearningResources for Search"""
+
+    topics = serializers.SerializerMethodField()
+
+    runs = OSLearningResourceRunSerializer(read_only=True, many=True, allow_null=True)
+
+    def get_topics(self, instance):
+        """Return the list of topics"""
+        return [topic.name for topic in instance.topics.all()]
```

Comment:
Ditto - is it possible to have `topics` in the same format across both search and learning_resources?